UPGRADE-4.2.md

```diff
@@ -179,6 +179,7 @@ FrameworkBundle
  * The `Templating\Helper\TranslatorHelper::transChoice()` method has been deprecated, use the `trans()` one instead with a `%count%` parameter.
  * Deprecated support for legacy translations directories `src/Resources/translations/` and `src/Resources/<BundleName>/translations/`, use `translations/` instead.
  * Support for the legacy directory structure in `translation:update` and `debug:translation` commands has been deprecated.
+ * The `GetSetMethodNormalizer` class correctly ignores the attributes specified in "ignored_attributes".
```

Comment:
we don't document bug fixes in this file, should be removed

Reply:
fixed